app/lib/pre_assembly/batch.rb

```diff
@@ -25,8 +26,8 @@ class Batch
              :staging_style_symlink,
              to: :batch_context
 
-    def initialize(batch_context, file_manifest: nil)
-      @batch_context = batch_context
+    def initialize(job_run, file_manifest: nil)
```

Comment:
The naming of things in Preassembly is challenging already, but I just want to point out that it seems a little weird to instantiate a Batch object with a JobRun instead of a BatchContext.